cmd/protoc-gen-crd/main.go

```diff
@@ -91,9 +107,26 @@ func generate(request *plugin.CodeGeneratorRequest) (*plugin.CodeGeneratorRespon
 		m,
 		descriptionConfiguration,
 		enumAsIntOrString)
-	return g.generateOutput(filesToGen)
+
+	for outputFileName, files := range channelOutput {
+		// TODO (whgriffi): fix the return to generate multiple files. At this time only the first file in the list is returned
```

Comment:
This is because https://github.com/istio/tools/pull/2797/files#diff-078824a78371270b3e5dabe3220d17cd0d520df4aefda21ac262a72d80137ba5R162 is invoking `generateSingleFileOutput`.